commerce-shipment-web/src/main/java/com/liferay/commerce/shipment/web/internal/display/context/BaseCommerceShipmentDisplayContext.java

```diff
@@ -257,11 +257,11 @@ protected String getDisplayStyle(
 	}
 
 	protected final ActionHelper actionHelper;
-	protected final CPRequestHelper cpRequestHelper;
 	protected final HttpServletRequest httpServletRequest;
 	protected final LiferayPortletRequest liferayPortletRequest;
 	protected final LiferayPortletResponse liferayPortletResponse;
 	protected final PortalPreferences portalPreferences;
+	protected final CommerceShipmentRequestHelper requestHelper;
```

Comment:
`CommerceShipmentRequestHelper requestHelper` should be `CommerceShipmentRequestHelper commerceShipmentRequestHelper`